.openpublishing.redirection.json

```diff
@@ -882,6 +894,11 @@
             "source_path": "docs/csharp/getting-started/with-visual-studio.md",
             "redirect_url": "/dotnet/core/tutorials/with-visual-studio"
         },
+        {
+            "source_path": "docs/csharp/how-to/safely-cast-using-pattern-matching-is-and-as-operators.md",
+            "redirect_url": "/dotnet/csharp/fundamentals/object-oriented/tutorials/safely-cast-using-pattern-matching-is-and-as-operators.md"
```

Comment:
Are these articles too deep? Won't they suffer from SEO problems because of that? I can't find the guidance on that though, I remember @mairaw talking about that a year or two ago.

Reply:
That's a good point. I'll move to one "Tutorials" node under Fundamentals.  I'll add that to this PR to avoid additional redirects later.

Reply:
The internal guidance is here: https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=main

"URLs must be consistent, hierarchical, and as shallow as possible to perform well in search. The more elements (folders) in the URL path, the harder for a search engine to crawl the content. Try to limit URLs to 2-3 levels deep from the base URL https://docs.microsoft.com/{lang-locale}/."

Reply:
For the C# guide, I can keep it to 4. 3 is really too flat for all that content.